### PR TITLE
Portfolio: finished mobile version.

### PR DESCRIPTION
**Pull request by Zafron047**
- [ ]  Mod Contact form

- [ ]  Fix contact form button color:

- Hover
- Active

### CODE REVIEW

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @Zafron047 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌

You have submitted a project with a readme file that is not detailed. Kindly edit and customize the install and run tests subsections of your readme with the project information for it to be professional ☺️. In summary the default guidelines should be customized.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Zafron047** [comment]
@AlphaNtihinduka Thanks for the advice. I have made the relevant changes. 😃

File: style.css [hunk 1]
```diff
@@ -389,12 +392,18 @@ hr {
...
 .contact-align hr {
   margin: 0 auto;
   margin-bottom: 8px;
   width: 125px;
   height: 5px;
-  background-color: black;
+  background-color: #fff;
   margin-top: 36px;
   border-radius: 10px;
   border: none;
```
Comment (ghost) [inline style.css:409]: - [ ] Good work on your contact form but there are some aspects of the design you can improve on. 
  - Reduce the spacing between the `input elements ` to match the original design

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Zafron047) [reply style.css:409]: @frank1738 Thanks for the review. I tried to follow the hard measurement given in Figma which was 20px. While I gave 15px and was a bit sceptic about this if not following the exact measurement might become an issue.

I agree that the gaps was not pleasant to look at. I have applied the measurement "11px" now it is look alike. And I am thankful for this advice now it is looking more nearer to the actual design given in figma.

😄

Comment (Olamarx) [reply style.css:409]: > * [ ]  Good work on your contact form but there are some aspects of the design you can improve on.
>   
>   * Reduce the spacing between the `input elements ` to match the original design
> 
> image
> 
> ### Checkout the original design
> image


- [ ] Kindly note that the previous reviewer was talking about height, not color. To simplify things, I have helped checked further, I noticed there is a <br> in between the inputs, when I removed the <br> tags, the spaces reduced to what the previous reviewer noticed so I would just recommend you remove the <br> tags and we are good to go.

### When I removed the `<br>`
image


### When the `<br>` was there
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters passed ✔️
- Professional readme file ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zafron047** [comment]
@Olamarx thanks a lot for the thorough inspection. After removing the `<br>` tags I could finally apply figma given [20px gap] gap measurement. And now it is looking good.

I hope the next review passes cause I have one last review left to submit.

I appreciate your effort.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Zafron047 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Zafron047) [reply style.css:409]: Fixed.

**Comment by Zafron047** [comment]
@Owusu-Desmond Thanks a lot for the review. 😃

**Pull request merged**